src/localsocketcontroller.cpp

```diff
@@ -45,37 +53,34 @@ LocalSocketController::LocalSocketController() {
   connect(m_socket, &QLocalSocket::readyRead, this,
           &LocalSocketController::readData);
 
+  m_initializingInterval = CONNECTION_RETRY_INITIAL_MSEC;
   m_initializingTimer.setSingleShot(true);
   connect(&m_initializingTimer, &QTimer::timeout, this,
           &LocalSocketController::initializeInternal);
 }
 
 LocalSocketController::~LocalSocketController() {
   MZ_COUNT_DTOR(LocalSocketController);
```

Comment:
Would clearAllTimeouts put us in an incorrect state if the daemon was delayed and then starts responding to previous responses?

Reply:
There shouldn't be anything in the daemon that blocks for any significant amount of time (everything should be handled async), so it would be an error if the daemon were to block long enough to trigger one of these timeouts.

Also, `clearAllTimeouts()` is used here under two conditions:
1. To cleanup and garbage collect when running the destructor, and
2. When handling an `errorOccurred()` signal, and we are about to try and reconnect. We don't want timeouts from the previous connection to interfere with the re connection attempt.